tests/object.cpp

```diff
@@ -923,6 +923,18 @@ TEST_CASE("object") {
         REQUIRE(obj.obj().get<String>(col_pk_str) == "value");
     }
 
+    SECTION("create null and 0 primary keys for Int types") {
```

Comment:
Shouldn't we add unit tests for all the primary key types?

Reply:
Added a test with ObjectId as primary key. We have one for string.